<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectUserLogin</class>
 <widget class="QDialog" name="selectUserLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboOwner">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>300</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboOficina">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>300</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelOwner">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Owners:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOficina">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Oficinas:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>704</x>
     <y>510</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableUser">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>800</width>
     <height>350</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::DashLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column/>
   <column/>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboOficina</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>selectUserLogin</receiver>
   <slot>showUsers()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>selectUserLogin</receiver>
   <slot>aceptar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>758</x>
     <y>512</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboOwner</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>selectUserLogin</receiver>
   <slot>showOficinas()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboOwner</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>selectUserLogin</receiver>
   <slot>showUsers()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectOwner(int)</slot>
  <slot>selectOficina()</slot>
  <slot>aceptar()</slot>
  <slot>showUsers()</slot>
  <slot>showOficinas()</slot>
 </slots>
</ui>
